LICENSE

```diff
@@ -0,0 +1,26 @@
+MIT License
+
+Copyright (c) 2016 Peter Elger
+Copyright (c) 2016 David Mark Clements
+Copyright (c) 2016 Dead McDonnell
+Copyright (c) 2016 Matteo Collina
```

Comment:
We should probably call it the "mu collaborators", and list everyone in a separate file.